Sources/GameMP/Computer.cpp

```diff
@@ -246,7 +246,7 @@ void LastUnreadMessage(void)
 // go to next/previous message
 void PrevMessage(void)
 {
-  if (_iActiveMessage<_acmMessages.Count()==0) {
+  if (_iActiveMessage < _acmMessages.Count()==0) { // FIXME: DG: what is this about?
```

Comment:
This seems to be equivalent to `if ( ! (_iActiveMessage < _acmMessages.Count()) )` which seems to make sense.. - any objections?